src/Vat.php

```diff
@@ -24,10 +29,9 @@ public function countryInEurope(string $countryCode): bool
         return $this->countries->isCountryValid($countryCode) && $this->countries->isCountryInEu($countryCode);
     }
 
-    public function europeanVatRate(string $vatNumber, string $countryCode): float
+    public function europeanVatRate(string $countryCode, ?DateTime $date = null): float
     {
-        // TODO: Implement
-        return 0.0;
+        return $this->vatRateResolver->getDefaultVatRateForCountry($countryCode, $date) ?? 0.0;
```

Comment:
I as an implementing party would like to be able to decide the default rate when none can be resolved.

Reply:
Fair point, I will change this.

Reply:
Fixed it.